File: prak3_php/Processor.php
<?php

class Processor{

    private $name;
    private $price;

    //konstruktor
    function __construct($name, $price){
        $this->name = $name;
        $this->price = $price;
    }

    //set
    public function setName($name){
        $this->name = $name;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    //get
    public function getName(){
        return $this->name;
    }

    public function getPrice(){
        return $this->price;
    }

    //destruktor
    function __destruct(){
    }

}


?>

File: prak3_python/Processor.py
class Processor:

    def __init__(self, name, price):
    #konstruktor
        self.name = name
        self.price = price

    #set
    def setName(self, name):
        self.name = name

    def setPrice(self, price):
        self.price = price

    #get
    def getName(self):
        return self.name

    def getPrice(self):
        return self.price
